firebase-firestore/src/main/java/com/google/firebase/firestore/local/IndexManager.java

```diff
@@ -74,6 +75,18 @@
   /** Returns all configured field indexes. */
   Collection<FieldIndex> getFieldIndexes();
 
+  /** Returns whether we can serve the given target from an index. */
+  boolean canServeFromIndex(Target target);
+
+  /**
+   * Iterates over all field indexes that are used to serve the given target, and returns the least
+   * recent offset of them all. Asserts that the target can be served from index.
+   */
+  IndexOffset minOffset(Target target);
```

Comment:
All Java methods should start with a verb: getMinOffset()

You should also align the two comments on their usage of "lowest" and "least recent" (which contrasts min).